app/views/shared/_personal_key_input.html.erb

```diff
@@ -0,0 +1,4 @@
+<input aria-label="<%= t('forms.personal_key.confirmation_label') %>"
+       autocomplete="off" class="col-12 mb4 border-dashed field monospace personal-key caps"
+       maxlength="<%= PersonalKeyFormatter.code_length %>" name="personal_key"
+       pattern="^<%= PersonalKeyFormatter.regexp_string %>$" required="" spellcheck="false" type="text" />
```

Comment:
I would just put all these extra attributes on their own lines, I think it's more readable

Reply:
Good call, I waffled a bit on it, but the readability is a better choice